test/run.sh

```diff
@@ -155,6 +155,17 @@ test_builder_node_version() {
   fi
 }
 
+test_nss_wrapper() {
+  local run_cmd="echo 'danbev:x:1000:1000:danbev test:/home/danbev:/bin/false' > passwd && LD_PRELOAD=libnss_wrapper.so NSS_WRAPPER_PASSWD=passwd NSS_WRAPPER_GROUP=group getent passwd danbev"
+  local expected="Error"
+  echo "Checking nss_wrapper ..."
+  out=$(docker run ${BUILDER} /bin/bash -c "${run_cmd}")
+  if ! echo "${out}" | grep -q "${expected}"; then
+    echo "ERROR[/bin/bash -c "${run_cmd}"] '${out}'"
+    return 1
+  fi
```

Comment:
Would it be better to explicitly return 0 in the success condition?

Reply:
Yeah, that might be better. I'll take a closer look at this tomorrow. I was a little rushed as I ran out of time today.